packages/discovery-react-components/src/components/SearchFacets/components/FacetsGroups/CategoryFacetsGroup.tsx

```diff
@@ -29,7 +29,7 @@ interface CategoryFacetsGroupProps {
   /**
    * Callback to handle changes in selected facets
    */
-  onChange: (selectedFacets: SelectedFacet[]) => void;
+  onCollapsibleFacetsGroup: (selectedFacets: SelectedFacet[]) => void;
```

Comment:
Following the naming convention you're using in other files, wouldn't this be `onCollapsibleFacetsGroupChange`?